jenkins-builds/7.0.UP02-20.11-dom-mc

```diff
@@ -23,6 +23,8 @@
  hub-2.14.2.eb                                          --set-default-module
  h5py-2.10.0-CrayGNU-20.11-python3-parallel.eb          --set-default-module
  h5py-2.10.0-CrayGNU-20.11-python3-serial.eb
+ h5py-3.2.1-CrayGNU-20.11-python3-parallel.eb
+ h5py-3.2.1-CrayGNU-20.11-python3-serial.eb
```

Comment:
```suggestion
 h5py-3.2.1-CrayGNU-20.11-python3-parallel.eb          --set-default-module
 h5py-3.2.1-CrayGNU-20.11-python3-serial.eb
```

Reply:
I suggest to remove the old release on Dom, so we move forward when running `UpdateEB`.